crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -215,27 +217,17 @@ impl<'flox> Publish<'flox, NixAnalysis> {
     }
 
     /// Write snapshot to catalog and push to origin
-    pub async fn push_catalog(self) -> Result<(), PublishError> {
-        let url = self.publish_ref.clone_url();
-        let repo_dir = tempfile::tempdir_in(&self.flox.temp_dir)
-            .unwrap()
-            .into_path(); // todo catch error
-        let catalog = <Git as GitProvider>::clone(url, &repo_dir, false)
-            .await
-            .unwrap(); // todo: catch error
-
-        if catalog.list_branches().await.unwrap() // todo: catch error
-            .into_iter().any(|info| info.name == "catalog")
-        {
-            catalog.checkout("catalog", false).await.unwrap(); // todo: catch error
-        } else {
-            todo!();
-            // catalog.checkout("catalog", true).await.unwrap(); // todo: catch error
-
-            // catalog.set_upstream("origin", "catalog").await.unwrap();  // todo: implement
-            //                                                               todo: catch error
+    pub async fn push_snapshot(&self) -> Result<(), PublishError> {
```

Comment:
Hmm yeah we probably need a better name. Snapshot doesn't really say what it is either - it could be a snapshot of anything. No need to bikeshed here though